Add the `osu! wiki` article, information on `Maintenance`, and `List of maintainers` (#6062)
ppy/osu-wiki: wiki/osu!_wiki/Maintenance/List_of_maintainers/en.md

# List of the osu! wiki maintainers

> The maintainers look after the [issue tracker](https://github.com/ppy/osu-wiki/issues), review and edit the pull requests from contributors, and merge them into the `master` branch of the repository, effectively making the changes public.

Reviewer: ```suggestion
The maintainers regularly look after the [issue tracker](https://github.com/ppy/osu-wiki/issues) and review pull requests from contributors. Contrary to other contributors, they can edit pull requests and merge them into the `master` branch of the repository, effectively making the changes public.
```

> The maintainers regularly look after the [issue tracker](https://github.com/ppy/osu-wiki/issues) and review pull requests from contributors. Apart from other contributors, they can edit pull requests and merge them into the `master` branch of the repository, effectively making the changes public.

Reviewer: ```suggestion
The maintainers regularly look after the [issue tracker](https://github.com/ppy/osu-wiki/issues) and review pull requests from contributors. Contrary to other contributors, they can edit pull requests and merge them into the `master` branch of the repository, effectively making the changes public.
```

> The maintainers regularly look after the [issue tracker](https://github.com/ppy/osu-wiki/issues) and review pull requests from contributors. Contrary to other contributors, they can edit pull requests and merge them into the `master` branch of the repository, effectively making the changes public.

Author: this shouldn't require serial comma

> If you need help with making a contribution or merging your pull request, talk to one of the maintainers in the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server) (all of them speak English).

Reviewer: ```suggestion
If you need help with making a contribution or want a review on your pull request, send a message in the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server) and a maintainer or another contributor will reply back (all maintainers know English).
```

> If you need help with making a contribution or want a review on your pull request, message in the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server) and a maintainer or another contributor will reply back (all maintainers know English).

Reviewer: ```suggestion
If you need help with making a contribution or want a review on your pull request, send a message in the `#osu-wiki` channel of the [osu!dev Discord server](/wiki/osu!dev_Discord_server) and a maintainer or another contributor will reply back (all maintainers know English).
```

| ![][flag_PL] [TPGPL](https://osu.ppy.sh/users/3944705) | [TPGPL](https://github.com/TPGPL) | **Active** | Polish |
| ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416) | [WalterToro](https://github.com/WalterToro) | Inactive | Spanish |

## The news team

Curators, editors, and approvers of all the [newsposts](https://osu.ppy.sh/home/news) that appear on the website.

| osu! profile | GitHub profile |
| :-- | :-- |
| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | [Ephemeralis](https://github.com/Ephemeralis) |
| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | [peppy](https://github.com/peppy) |
| ![][flag_US] [pishifat](https://osu.ppy.sh/users/3178418) | [pishifat](https://github.com/pishifat) |

## Past maintainers

This list includes maintainers of the MediaWiki setup, which the osu! wiki [used to have](/wiki/History_of_osu!/osu!_wiki):

- ![][flag_RU] [\[Dellirium\]](https://osu.ppy.sh/users/519032)
- ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370)
- ![][flag_PL] [Galkan](https://osu.ppy.sh/users/169570)
- ![][flag_PH] [Nathanael](https://osu.ppy.sh/users/2295078)
- ![][flag_PL] [Ukami](https://osu.ppy.sh/users/820865)

> [flag_AR]: /wiki/shared/flag/AR.gif
> [flag_AU]: /wiki/shared/flag/AU.gif
> [flag_CH]: /wiki/shared/flag/CH.gif
> [flag_CL]: /wiki/shared/flag/CL.gif
> [flag_DE]: /wiki/shared/flag/DE.gif
> [flag_FR]: /wiki/shared/flag/FR.gif
> [flag_MX]: /wiki/shared/flag/MX.gif
> [flag_NZ]: /wiki/shared/flag/NZ.gif
> [flag_PH]: /wiki/shared/flag/PH.gif
> [flag_PL]: /wiki/shared/flag/PL.gif
> [flag_RU]: /wiki/shared/flag/RU.gif
> [flag_US]: /wiki/shared/flag/US.gif

Reviewer 2: ```suggestion
[flag_AR]: /wiki/shared/flag/AR.gif "Argentina"
[flag_AU]: /wiki/shared/flag/AU.gif "Australia"
[flag_CH]: /wiki/shared/flag/CH.gif "Switzerland"
[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
[flag_FR]: /wiki/shared/flag/FR.gif "France"
[flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
[flag_NZ]: /wiki/shared/flag/NZ.gif "New Zealand"
[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
[flag_RU]: /wiki/shared/flag/RU.gif "Russian Federation"
[flag_US]: /wiki/shared/flag/US.gif "United States"
```

## The wiki team

<!-- internal reference: https://github.com/orgs/ppy/teams/wiki/members -->

| osu! profile | GitHub profile | Status | Additional languages |
| :-- | :-- | :-- | :-- |
| ![][flag_US] [clayton](https://osu.ppy.sh/users/3666350) | [cl8n](https://github.com/cl8n) | Inactive | - |
| ![][flag_AU] [Ephemeral](https://osu.ppy.sh/users/102335) | [Ephemeralis](https://github.com/Ephemeralis) | **Active** | - |
| ![][flag_US] [Joehu](https://osu.ppy.sh/users/8549835) | [Joehuu](https://github.com/Joehuu) | **Active** | - |
| ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656) | [juankristal](https://github.com/juankristal) | Inactive | Spanish |
| ![][flag_DE] [Loctav](https://github.com/Loctav) | [Loctav](https://github.com/Loctav) | Inactive | German |
| ![][flag_US] [MegaApple_Pi](https://osu.ppy.sh/users/2148208) | [MegaApplePi](https://github.com/MegaApplePi) | Inactive | - |
| ![][flag_DE] [Okoratu](https://osu.ppy.sh/users/1623405) | [Okorin](https://github.com/Okorin) | Inactive | German |

> | ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | [peppy](https://github.com/peppy) | Inactive | - |

Author: that didn't cross my mind, thanks
